### ISSUE TITLE
Who maintains a certain repository?

### ISSUE THREAD

**Issue by Martin-P** [zendframework/zf2#7607]
In this post https://github.com/zendframework/zf2/issues/7072#issuecomment-68006289 @Ocramius said the following:
> Also, we will be able to transfer ownership of the single packages to the various maintainers, allowing them to tag/release and merge independently from the framework decisions.

Because ZF2 is split up in single packages now, perhaps the readme files in the corresponding repositories can be updated with the maintainer names? This creates clarity in who is responsible for maintaining a repository.

**Comment by Maks3w** [comment]
Exactly what do you want. Do you want to know who is the maintainer for to send him a pack of beers?

https://github.com/zendframework/zf2/wiki/Component-Maintainers-for-ZF-2.0

**Comment by Ocramius** [comment]
I actually agree with @Martin-P here: having a clear reference of the
maintainers in the repository is very useful to users of the package.
Additionally, whether to add the name or not should really be up to each
maintainer (some don't really like being that exposed).

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by gianarb** [comment]
I proposed this idea but it's frozen :)
https://github.com/zendframework/zf-web/pull/161

**Comment by Martin-P** [comment]
> Do you want to know who is the maintainer for to send him a pack of beers?

If you want some beers, no problem --> :beers:  :smile: 

At this moment it looks like all issues in every repo are handled by @weierophinney, @Ocramius and @Maks3w, but I believe the ZF2 maintainers team is more than just the 3 of you? In case of specific questions I think it is easier to ask the maintainer of a repo about his/her opinion. I think it might also be helpful in keeping up with issues/PR's. Many hands make light work.

**Comment by gianarb** [comment]
@Martin-P for president! :+1: :beers: 

I agree with you.. if the community is big in this moment it is sleeping :)
 We should find a good method to wake it and this can be a good opportunity

**Comment by Maks3w** [comment]
You forgot to mention @zendframework/community-review-team

**Comment by gianarb** [comment]
@Maks3w can you reopen this issue? This is a good observation

**Comment by gianarb** [comment]
Thanks @Ocramius! :)

**Comment by Martin-P** [comment]
Any news on this? Or is it just considered a "good observation" and that's it? :wink:

**Comment by gianarb** [comment]
I can only consider :+1:

**Comment by mwillbanks** [comment]
@Martin-P I will bring this up with Matthew when I have a second today.

**Comment by Martin-P** [comment]
@mwillbanks Already had a second the last 26 days? :wink:

**Comment by mwillbanks** [comment]
Eeek ;)

**Comment by weierophinney** [comment]
Unfortunately, due to the large number of notifications I get, I'm literally just seeing this today.

The wiki page referenced is an old one, dating to the initial 2.0 development. We need to update it. What I'd like to do is:

- Put it on either the [maintainers wiki](https://github.com/zendframework/maintainers/wiki) or as a markdown document in that repository; essentially, something we can link to that's evergreen.
- Use github usernames (via `@username` notation) in the page.

Anybody want to take an initial stab at this?

**Comment by gianarb** [comment]
I started same stuff into the site.. I can rewrite in into the maintainers
repo (markdown document) :)

I work about this PR

2015-09-01 20:10 GMT+02:00 weierophinney <notifications@github.com>:

> Unfortunately, due to the large number of notifications I get, I'm
> literally just seeing this today.
>
> The wiki page referenced is an old one, dating to the initial 2.0
> development. We need to update it. What I'd like to do is:
>
>    - Put it on either the maintainers wiki
>    <https://github.com/zendframework/maintainers/wiki> or as a markdown
>    document in that repository; essentially, something we can link to that's
>    evergreen.
>    - Use github usernames (via @username notation) in the page.
>
> Anybody want to take an initial stab at this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zf2/issues/7607#issuecomment-136815259>.
>



-- 
Gianluca Arbezzano
www.gianarb.it

**Comment by marc-mabe** [comment]
Hi guys
We could also use the `composer.json` file of each repo for this and implement a small "5 liner" to update a table with all of us.

**Comment by weierophinney** [comment]
@marc-mabe That sounds painful; it'd be easy for it to go out of scope over time, and then we'd have N+1 places to keep track of (where N is the number of components, and the "1" is the maintainers repo file). Let's keep it to 1. :smile:

**Comment by marc-mabe** [comment]
@weierophinney I don't think that keeping it in `composer.json` will go out of scope because every maintainer has (or should have) access to it. On the other hand it needs someone else to keep track of it. And having it in `composer.json` means it is directly within the maintained repository and can be fined by every one as of this is going to be the quasi standard and it will already be used for some other tools like packagist. But I totally agree having it visible in one position is very handy but it should be very simply writing a small script to update it automatically.

**Comment by Maks3w** [comment]
I don't like the idea of track maintainers in composer.json. Composer.json is frozen after each release and it's imposible to backport any change. As far as I see people of this thread wants an up to date list, composer.json is a long time ago these were the maintainers

**Comment by marc-mabe** [comment]
@Maks3w Sure it's frozen for each release but it list the maintainer that was one on this release. The up-to-date maintainer is in the current master/develop branch

**Comment by ThaDafinser** [comment]
:+1:  for being in composer.json.

Then i could include it here as well http://thadafinser.github.io/zf-status/
